kahuna/public/js/sentry/sentry.js

```diff
@@ -3,20 +3,18 @@ import raven from 'raven-js';
 
 export var sentry = angular.module('sentry', []);
 
-// TODO: Add user information
-// TODO: Catch angular errors nicely
-
 sentry.factory('sentryEnabled', ['sentryDsn', function(sentryDsn) {
     return angular.isString(sentryDsn);
 }]);
 
-sentry.factory('sentry', [function() {
-
-    function trigger(errorMessage, extra) {
-        raven.captureException(new Error(errorMessage), { extra });
-    }
-
-    return { trigger };
+// TODO: Alternatively could investigate angular-raven
+sentry.config(['$provide', function ($provide) {
+    $provide.decorator('$exceptionHandler', ['$delegate', function ($delegate) {
```

Comment:
Good catch, it is.

I was hoping to be able to do something clever where only uncaught HTTP error would bubble, unfortunately **all** throw exceptions (as opposed to returning `$q.reject(...)`) bubble to the global error handler no matter whether you catch them or not, and that includes HTTP errors.

I've added a fix in another commit.